conanfile.txt

```diff
@@ -9,8 +9,4 @@ spdlog/1.9.2
 # sdl2/2.0.1
 
 [generators]
-cmake_find_package
-cmake
-gcc
-txt
-visual_studio
\ No newline at end of file
+cmake_find_package_multi
```

Comment:
Shouldn't we have both?
```suggestion
cmake_find_package_multi
cmake_find_package
```

Reply:
No! Actually. I say this strongly because I got caught in an infinite mental loop of trying to have more than one enabled.

Just the `cmake_find_package_multi` is all we need for it to work with every generator. If you enable both, then the config files for the multi get overwritten by the regular one, and if you enable only the regular one then MSVC cannot find all packages when you dynamically change the build type from inside of VisualStudio. 

If you enable only `cmake_find_package_multi` then everyone plays nicely (even just plain Makefiles)

Reply:
Thanks for the explanation!